src/test/java/bio/terra/app/controller/SnapshotAccessRequestApiControllerTest.java

```diff
@@ -147,6 +145,32 @@ void testGetSnapshotRequest() throws Exception {
             IamAction.GET);
   }
 
+  @Test
+  void testGetSnapshotRequestDetails() throws Exception {
+    var model = SnapshotBuilderTestData.createSnapshotAccessRequestModel(SNAPSHOT_ID);
+    var expectedResponse =
+        model.generateModelDetails(
+            SnapshotBuilderTestData.SETTINGS,
+            Map.of(SnapshotBuilderTestData.SNAPSHOT_BUILDER_COHORT_CONDITION_CONCEPT_ID, "unused"));
```

Comment:
For the purposes of this test, there is no need to generate a real summary object, and doing it creates a dependency on code not being tested here. Instead I would use a dummy response object.
```suggestion
    var expectedResponse = new SnapshotAccessRequestDetailsResponse().summary("summary");
```